src/neuroconv/basedatainterface.py

```diff
@@ -146,15 +143,15 @@ def run_conversion(
         overwrite : bool, default: False
             Whether to overwrite the NWBFile if one exists at the nwbfile_path.
             The default is False (append mode).
-        backend : "hdf5", optional
-            The type of backend to use when writing the file.
-            If a `backend_configuration` is not specified, the default type will be "hdf5".
-            If a `backend_configuration` is specified, then the type will be auto-detected.
-        backend_configuration : HDF5BackendConfiguration, optional
-            The configuration model to use when configuring the datasets for this backend.
-            To customize, call the `.get_default_backend_configuration(...)` method, modify the returned
-            BackendConfiguration object, and pass that instead.
-            Otherwise, all datasets will use default configuration settings.
+        backend : {'hdf5', 'zarr'}, default: 'hdf5'
+            The storage backend for the NWB file.
+            If `backend_configuration` is provided, this value is overridden by the configuration's backend type.
+        backend_configuration : HDF5BackendConfiguration or ZarrBackendConfiguration, optional
+            Configuration settings for the chosen backend. If not provided, default settings are used.
+            To customize, obtain the default configuration using `.get_default_backend_configuration(...)`,
+            modify it, and pass the modified configuration here.
+        conversion_options : dict, optional
```

Comment:
Got you, thanks.
```suggestion
        **conversion_options : dict, optional
```
So you mean it should be like this? 

Not sure about the typing.

Reply:
IDK that we need to type it in the base class

It's mostly to allow pass-through from child classes that would define and annotate specific arguments

Reply:
OK!